platforms/ios/lib/WysiwygComposer/Sources/WysiwygComposer/Components/WysiwygComposerView/WysiwygComposerViewModel.swift

```diff
@@ -560,15 +646,23 @@ private extension WysiwygComposerViewModel {
             plainTextContent = NSAttributedString()
         }
     }
-
+    
     /// Reconciliate the content of the model with the content of the text view.
     func reconciliateIfNeeded() {
         do {
             guard !textView.isDictationRunning,
                   let replacement = try StringDiffer.replacement(from: attributedContent.text.htmlChars,
-                                                                 to: textView.attributedText.htmlChars) else {
+                                                                 to: textView.attributedText.htmlChars)
+            else {
```

Comment:
`swiftformat` doesn't complain about this?

Reply:
Doesn't appear to, want the else moved up to the previous line? happy to do so.